choosing-packages.md

```diff
@@ -0,0 +1,49 @@
+# Selecting packages and dependencies
+
+We use a lot of packages and dependencies in our services. If you have a problem to solve or a feature to implement, it can often be easier to use an established package than to hand-crank your own solution - provided that package meets certain criteria.
+
+When you are picking a package or a dependency to include in your project, there are a number of things to consider:
+
+## What have other Defra services used?
+
+Is there an agreed-upon standard package for this purpose? If there is, use it.
+
+For example, Hapi is our Node.js framework of choice. So if you're starting a new Node project and need a web framework, you should use Hapi over alternatives like Express.
+
+Even if there's no official standard, you should still consider what similar services have used in the past. Using the same packages across services makes our codebases more consistent and easier to maintain. You may also find it quicker to put the package into use because you can reuse code from our other services, or talk to developers who already know how to use it.
+
+## What are the most popular solutions?
+
+If you're looking for a package which performs a common function (like logging or user authentication), there will probably be a lot of packages that could meet your needs.
+
+When in doubt, go for the most commonly used package, especially if it's so popular as to become the industry standard.
```

Comment:
Can we say how to identify what is "the most commonly used"?

Reply:
Added some detail in https://github.com/DEFRA/software-development-standards/pull/7/commits/79021a9b5d59bff83f5a590010ed2fe052fbf690